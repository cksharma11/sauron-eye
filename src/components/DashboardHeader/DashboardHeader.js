import React from "react";

const DashboardHeader = ({ headings }) => {
  return (
    <tr>
      {headings.map(heading => (
        <th key={heading}>{heading}</th>
      ))}
    </tr>
  );
};

export default DashboardHeader;
