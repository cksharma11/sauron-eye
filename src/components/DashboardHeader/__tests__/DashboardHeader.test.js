import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import DashboardHeader from "../DashboardHeader";

configure({ adapter: new Adapter() });

describe("DashboardHeader", () => {
  it("should render properly", () => {
    const wrapper = shallow(<DashboardHeader headings={["A", "B", "C"]} />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
