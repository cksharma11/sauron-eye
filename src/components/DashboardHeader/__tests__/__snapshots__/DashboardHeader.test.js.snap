// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardHeader should render properly 1`] = `
"<tr>
  <th>
    A
  </th>
  <th>
    B
  </th>
  <th>
    C
  </th>
</tr>"
`;
